doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -58,113 +59,109 @@ sdk\<service name>\<package name>\CHANGELOG.md
 sdk\<service name>\<package name>\<package name>.sln
 ```
 
-- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate
+- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate. It is usually your service name in REST API specifications. For instance, if the API spec is under `specification/storage`, the service would normally be `storage`.
 - `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
 
-### Use Cadl as Input
+### Use Cadl as Input API spec
 
 #### Create Cadl project
   
   You can download existing cadl project from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow following steps to create a new cadl project.
 
   ***Initialize Cadl Project***
 
-  Follow [Cadl Getting Start](https://github.com/microsoft/cadl/#using-node--npm) to initialize your Cadl project.
+  Follow [Cadl Getting Start](https://github.com/microsoft/typespec/#using-node--npm) to initialize your Cadl project.
```

Comment:
This leads them down the wrong path, they won't have to do any of these steps if they use `dotnet build /t:GenerateCode`

Reply:
hello @m-nash This chapter is to describe how to initialize a cadl project to write cadl files.

Reply:
This isn't something the `azure-sdk-for-net` repo needs to describe.  This is something the `azure-rest-api-specs` needs to describe.

We need a document which talks about how to generate the dotnet code.  This document can reference a `One time setup` step if needed for completeness but by the time you get to this document you have already written your cadl so we don't need to explain how to write your cadl.